internal/pkg/cli/svc_logs.go

```diff
@@ -99,6 +104,9 @@ func newSvcLogOpts(vars wkldLogsVars) (*svcLogsOpts, error) {
 		if err != nil {
 			return err
 		}
+		opts.newSvcDescriber = func() serviceDescriber {
+			return ecs.New(sess)
+		}
```

Comment:
oh! instead of this can we rename `initLogsSvc` to `initRuntimeClients` and do this inside:
```go
opts.ecs = ecs.New(sess)
```

Reply:
I could do that but this `initLogsSvc` is being used in job_logs as well and there it is just setting `opts.logsSvc`. Is it okay to still change the name to `initRuntimeClients`?